src/pudl/transform/eia860.py

```diff
@@ -568,14 +556,8 @@ def _core_eia860__plants(raw_eia860__plant: pd.DataFrame) -> pd.DataFrame:
             .replace(to_replace=["Y", "N", "NaN"], value=[True, False, pd.NA])
         )
 
-    p_df = pudl.helpers.convert_to_date(p_df)
-
-    p_df = clean_nerc(p_df, idx_cols=["plant_id_eia", "report_date", "nerc_region"])
-
-    p_df = (
-        pudl.metadata.classes.Package.from_resource_ids()
-        .get_resource("core_eia860__scd_plants")
-        .encode(p_df)
+    p_df = pudl.helpers.convert_to_date(p_df).pipe(
+        clean_nerc, idx_cols=["plant_id_eia", "report_date", "nerc_region"]
```

Comment:
Removed the encoding step because it's the last part of the transform, and now happens across the board in the harvesting process.